vuepress/docs/v7.0/docs/reference/databases.md

```diff
@@ -282,20 +281,19 @@ jdbc:postgresql://10.0.0.13:5432/my\_db
 
 #### Notes
 
-PostgreSQL behaves very similar to Oracle when it comes to how it
-associates a user with its own schema. The current username is used as a
-default schema/prefix to resolve tables. As with Oracle, Entando ensures
-that two users don’t have access to each other’s schemas.
+* PostgreSQL behaves like Oracle when it comes to user and schema association. The current username is used as a
+default schema/prefix to resolve tables. Entando ensures
+that two users don’t have access to the other’s schemas.
 
-## Skipping database preparation
+## Skipping Database Preparation
 
-When an Entando App is being deployed, there is an operator responsible for the entire deployment process. It also takes care of DB creation and preparation.
-If you already have a prepared DB (schemas, tables, etc.), you could skip schemas creation and DB preparation of the EntandoApp in order to speed up the deployment process.
+When an Entando Application is deployed, an operator is responsible for the entire process, including DB creation and preparation.
+If you already have a prepared DB (schemas, tables, etc.), you could skip the schema creation and DB preparation to speed up the deployment process.
 
-You can achieve this by specifying some properties for the EntandoApp component present in the `entandoapp.yaml` file.
+To achieve this, specify the pertinent properties for the EntandoApp component in the `entandoapp.yaml` file.
```

Comment:
of the EntandoApp

Reply:
i'm setting a record for thumbs down. why "for" and not "of"? rephrasing for clarity, it's "the properties of the EntandoApp component," not "the properties for the EntandoApp component"

noting that the yaml file is backticked here

Reply:
backticked because it's referencing a specifically defined file. 

Spending way too much time on this so just giving thumbs down. I think the return on time investment is not great

Reply:
definitely no return on investment